tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -139,8 +142,6 @@ def executor_kwargs(logger, test_type, test_environment, run_info_data,
     if test_type == "wdspec":
         options = {"args": []}
         if kwargs["binary"]:
-            if "webdriver_args" not in executor_kwargs:
-                executor_kwargs["webdriver_args"] = []
             executor_kwargs["webdriver_args"].extend(["--binary", kwargs["binary"]])
```

Comment:
Do we still need this line? When I remove it locally it doesn't make a difference. So what's the difference between executor args and browser args?

Reply:
We use this to write the configuration that gets passed in to the tests. So this will make a difference for tests that try to restart geckodriver itself, like some of the internal tests do.

Reply:
Hm, ok. I just run a simple wdspec test so that might be the reason why I didn't notice a difference. As such we should leave it here then.